### PR TITLE
Use yaml for config files

### PR DESCRIPTION
**Pull request by mvdan**
Note that the json->yaml conversion was automated via python. I only checked that the order of the fields was kept and that the files made sense, no more. Hoping the tests will still pass.

### CODE REVIEW

**Comment by mvdan** [comment]
In fact, some of the fields weren't kept in the right order. I can manually fix that if it's a problem.

**Comment by vishh** [comment]
A huge +1. This change helps keep configs across k8s and heapster repo consistent.

File: deploy/kube-config/google/heapster-service.json [hunk 1]
```diff
@@ -1,23 +0,0 @@
-{
```
Comment (vishh) [inline deploy/kube-config/google/heapster-service.json]: While you are here, should we symlink the service definitions instead of duplicating them?

**Comment by vishh** [comment]
The order of the fields does not matter IMO. From a brief glance, the PR LGTM.

File: deploy/kube-config/google/heapster-service.json (hunk 1 above)
Comment (mvdan) [inline deploy/kube-config/google/heapster-service.json]: What about having it as `deploy/kube-config/heapster-service.json`, if it applies to all different configs?

Comment (vishh) [inline deploy/kube-config/google/heapster-service.json]: +1

**Comment by vishh** [comment]
LGTM

**Pull request merged**